.eslintrc.js

```diff
@@ -28,5 +28,11 @@ module.exports = {
         tryExtensions: [".js", ".json", ".node", ".ts"],
       },
     ],
+    camelcase: [
```

Comment:
what is this for?

Reply:
<img width="650" alt="image" src="https://github.com/bcnmy/scw-contracts/assets/16562513/e13db1f2-83c2-4b1e-a03b-121fe179803b">

Prevents eslint from showing these errors.